requirements.txt

```diff
@@ -0,0 +1,2 @@
+-r ci/run_tests/requirements.txt
+-r Tests/ExamplesTest/requirements.txt
```

Comment:
I would put this file in the root directory of the project because it's only needed for tests

Reply:
Yes and I think that ci/run_tests should be moved to Tests no ? something like Tests/LiveReplay ?

Reply:
I don't think we need this file. We run `ci/run_tests` and `Tests/ExamplesTest` separately so we can install the python packages in separate steps before each run